### ISSUE TITLE
SVG using namespace-less prefix renders in browser, won't post

### ISSUE THREAD

**Issue by BooDoo** [BooDoo/CheapBotsTootSweet#5]
Try:

```svg
{
	"origin": "{svg #prefix##image##text#</svg>}",
        "prefix": "<svg width='800px' height='400px'>",
        "image": "<image xlink:href='https://placeimg.com/640/480/animals/image.jpg' width='300px' height='300px' preserveAspectRatio='xMidYMid slice'/>",
        "text": "<text text-anchor='middle' x='20%' y='40%' style='font: italic 40px serif; fill: red;'>TESTING</text>"
}
```

**FRONTEND**: Renders in browser, Firefox notes parsing error Chrome doesn't complain
**SEND_STATUS**: does not successfully Post using Post! button (FF and Chrome both)
**BACKEND**: scheduled posting impact still TBD

**Comment by BooDoo** [comment]
Firefox throws SVG parse error asking for namespace, Chrome shows no error.

Added 
`xmlns:xlink='http://www.w3.org/1999/xlink'`
and was able to post from both Chrome and Firefox.

Follow-up: 
  - [ ] verify backend impact
  - [x] update documentation
  - [ ] ~~force-add these attributes when absent?~~

**Comment by BooDoo** [comment]
changing to "enhancement" since invalid/partial SVG isn't technically our fault/problem.
Documentation definitely needs to be updated ASAP, though.

**Comment by BooDoo** [comment]
We might fix this for "free" as part of #7

**Comment by BooDoo** [comment]
With #7, we resolved blocking behavior of this issue.

Minimal SVG boilerplate is required and if prefix w/o namespace is applied to a standard supported attribute (e.g. `<image xlink:href...` instead of `<image href...`) the Tracery validator warning does show up, but preview renders and you can Post status and Save changes, so this is no big deal.
